src/builtins/alias.c

```diff
@@ -0,0 +1,98 @@
+/*
+** EPITECH PROJECT, 2022
+** mysh
+** File description:
+** alias builtins command
+*/
+
+#include <unistd.h>
+#include <string.h>
+#include <errno.h>
+#include <stdlib.h>
+#include "my_dico.h"
+#include "my_strings.h"
+#include "my_puts.h"
+#include "my_wordarray.h"
+#include "builtins.h"
+#include "mysh.h"
+#include "mysh_struct.h"
+
+list_t *my_alias_add(list_t *alias, char *name, char *value) {
+    char *new_name = NULL;
+    char *new_value = NULL;
+
+    new_name = my_strdup(name);
+    new_value = my_strdup(value);
```

Comment:
check if new_value == NULL || new_name == NULL
because it will seg with free(new_name) or other

Reply:
ok